app/Model/CoSetting.php

```diff
@@ -585,4 +605,33 @@ public function _ug400() {
       array('CoSetting.global_search_limit'=> DEF_GLOBAL_SEARCH_LIMIT)
     );
   }
+
+  /**
+   * Perform CoSetting model upgrade steps for version 4.1.0.
+   * This function should only be called by UpgradeVersionShell.
+   *
+   * @since  COmanage Registry v4.1.0
+   */
+
+  public function _ug410() {
```

Comment:
I don't think we need an upgradeShell function (which doesn't look like it's called anyway) since `CoSetting::lookupValue()` will use the default value (via `$defaultSettings`) if no value is found.